bellman/src/lib.rs

```diff
@@ -1,18 +1,16 @@
-extern crate ff;
-extern crate group;
-#[cfg(feature = "pairing")]
-extern crate pairing;
-extern crate rand_core;
 
-extern crate bit_vec;
-extern crate blake2s_simd;
-extern crate byteorder;
-extern crate futures;
+
+
+
+
+
+
+
+
 
 #[cfg(feature = "multicore")]
 extern crate crossbeam;
-#[cfg(feature = "multicore")]
-extern crate futures_cpupool;
```

Comment:
Why does this `multicore`-conditional `extern crate` disappear, but not the other two?

Reply:
`extern crate hex_literal` stays as expected because of the `#[macro_use]`.